tests/test_actor.py

```diff
@@ -103,7 +105,7 @@ def test_clears_cleanup_tasks() -> None:
     actor2.has_independent_cleanup_tasks(mocked_task_with_exception)
 
     actor1.cleans_up()
-    with pytest.raises(ValueError):
+    with pytest.raises(ValueError, match=test_msg):
```

Comment:
Does this validate the error msg in ValueError?

Reply:
Oh this one i actually liked a lot—this one _only_ matches a `ValueError` with the indicated message, which allows other `ValueError`s to bubble up.